test/integration/auth/rbac_test.go

```diff
@@ -513,6 +553,11 @@ func TestRBAC(t *testing.T) {
 			}
 
 			req, err := http.NewRequest(r.verb, s.URL+path, body)
+			if r.verb == "PATCH" {
+				// For patch operations, use the apply content type
+				req.Header.Add("Content-Type", "application/apply-patch+yaml")
```

Comment:
is this new here? is there an existing patch type for this?

Reply:
The test doesn't have any cases for patch yet, probably because it needs a different content type

Reply:
Instead of special casing patch, I could add content type as a part of the request struct in the test, but since they are always created without specifically naming the fields, it would mean adding ```, ""``` to all the existing requests

Reply:
I actually meant if there was a patch type like JSONPatchType/MergePatchType/StrategicMergePatchType we could reference.

Reply:
Ah I see what you mean. you're right I'll change it to types.ApplyPatchType

Reply:
Okay, it's fixed now

Reply:
unrelated to this, but I was surprised to not see both ApplyPatchJSONType and ApplyPatchYAMLType... is that planned/tracked somewhere?
cc @apelisse